types/tusk-mastodon/index.d.ts

```diff
@@ -0,0 +1,32 @@
+declare class TuskMastodonInstance {
+    constructor(options: TuskMastodonOptions);
+
+    put(path: string, params: GeneralObject): Promise<GeneralObject>;
```

Comment:
all these `params` seem to be optional parameter:
https://github.com/skullzarmy/Tusk/blob/master/lib/mastodon.js#L19 (etc)

Reply:
Fixed in b514bfa3bbd9b586228961070f40026c674c1396